src/app/news-type-filter.pipe.spec.ts

```diff
@@ -1,8 +1,83 @@
 import { NewsTypeFilterPipe } from './news-type-filter.pipe';
```

Comment:
This pipe is exactly the same as `news-filter.pipe.ts`
Seems like we could just delete one of them? @jeznorth

Reply:
Leave for now, however we should investigate where this overlap is and consolidate if necessary.